build/Katana.version.targets

```diff
@@ -11,7 +11,7 @@
     <BuildQuality>rtm</BuildQuality>
     <BuildDate>70129</BuildDate>
     <BranchSuffix></BranchSuffix>
-    <Eula>https://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</Eula>
+    <Eula>https://raw.githubusercontent.com/aspnet/AspNetKatana/v4.0.1/LICENSE.txt</Eula>
```

Comment:
Note v4.0.1 points to a branch for now. We'll change it to a tag when we release.